composer.json

```diff
@@ -11,27 +11,27 @@
         }
     ],
     "require": {
-        "php": ">=5.4",
-        "symfony/options-resolver": "~2.3|~3.0",
-        "psr/log": "~1.0"
+        "php": "^5.4 || ^7.0",
+        "symfony/options-resolver": "^2.3 || ^3.0",
+        "psr/log": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.7|~5.0",
-        "php-amqplib/php-amqplib": "~2.1",
-        "doctrine/common": "~2.3",
-        "doctrine/dbal": "~2.2"
+        "phpunit/phpunit": "^4.7 || ^5.0",
```

Comment:
I wonder if we should not remove the phpunit dependency, and use the global one on travis instead ? Or even suggest using the global one if we need to run the tests...